libqtile/utils.py

```diff
@@ -110,6 +110,43 @@ def hex(x):
     return '#%02x%02x%02x' % (int(r * 255), int(g * 255), int(b * 255))
 
 
+def has_transparency(colour):
```

Comment:
These can be typed using `ColorType`

Reply:
Indeed. Need to get better at typing.

Reply:
I think it's actually typed with `Union[ColorType, List[ColorType]]` as it can also accept list of colours (e.g. where a user has specified multiple colours for a gradient)